### ISSUE TITLE
Error when trying to Select themes, and all previously imported themes are now gone from the app.

### ISSUE THREAD

**Issue by lianesilva** [t1m0thyj/WinDynamicDesktop#492]
- WDD Version: 5.1.0 (UWP)
- OS Version: Windows 10, 21H2

Steps to Reproduce:

1. Right-click WinDynDesktop system tray icon
2. Click on "Select Theme..."

OR

1. Double-click .ddw file

Are you using the Microsoft Store version of WDD?: Yes

![image](https://user-images.githubusercontent.com/67926441/201465170-6afe0937-4c51-4178-88e5-936f73d0d837.png)

**Comment by t1m0thyj** [comment]
Please attach the log file from the path in the error message.

**Comment by aacar947** [comment]
[WinDynamicDesktop.dll.log](https://github.com/t1m0thyj/WinDynamicDesktop/files/10145900/WinDynamicDesktop.dll.log)
Apperntly I get the same error.

**Comment by t1m0thyj** [comment]
@aacar947 In the log file it has messages like the following:
```
[2022-12-03 06:57:09.058] System.IO.IOException: The process cannot access the file 'C:\Users\aacar\AppData\Local\Packages\38719TimothyJohnson.WinDynamicDesktop_rfarkse4rn21a\LocalState\themes\The_Cliffs.zip' because it is being used by another process.
[2022-12-03 07:00:46.743] System.IO.IOException: The process cannot access the file 'C:\Users\aacar\AppData\Local\Packages\38719TimothyJohnson.WinDynamicDesktop_rfarkse4rn21a\LocalState\themes\The_Lake.zip' because it is being used by another process.
```

This makes me wonder if there is another program on your system (perhaps antivirus software) that is holding the files open and has them locked. Could you navigate to these zip files in File Explorer, and click on "What's using this file?" to see what process might be the culprit?
![image](https://user-images.githubusercontent.com/22344007/205442094-b624fa34-ebf0-4c65-8558-b6cfe577b31a.png)

**Comment by lianesilva** [comment]
Sorry, keep forgetting to check GitHub for new notifications!
 
Here's my log file
[WinDynamicDesktop.dll.log](https://github.com/t1m0thyj/WinDynamicDesktop/files/10423837/WinDynamicDesktop.dll.log)

**Comment by lianesilva** [comment]
> 

Tried doing this, but I don't have any zipped files in that directory, all of my themes are unzipped.

**Comment by t1m0thyj** [comment]
There were some cases where WinDD attempted to extract the file while it was still locked for downloading. This error should be fixed in v5.3.0.
